dbt/seeds/row_counts.csv

```diff
@@ -0,0 +1,270 @@
+table_name,partition,row_count
+out_vcerare__hourly_available_capacity_factor,2019,27287400
+out_vcerare__hourly_available_capacity_factor,2020,27287400
+out_vcerare__hourly_available_capacity_factor,2021,27287400
+out_vcerare__hourly_available_capacity_factor,2022,27287400
+out_vcerare__hourly_available_capacity_factor,2023,27287400
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,1994,1188
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,1995,1203
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,1996,1143
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,1997,1112
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,1998,1113
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,1999,1073
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2000,1041
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2001,923
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2002,877
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2003,856
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2004,832
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2005,873
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2006,865
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2007,877
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2008,857
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2009,867
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2010,883
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2011,885
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2012,889
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2013,858
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2014,833
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2015,826
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2016,804
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2017,780
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2018,755
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2019,743
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2020,716
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2021,751
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2022,765
+out_ferc1__yearly_steam_plants_fuel_by_plant_sched402,2023,759
+out_eia923__boiler_fuel,2008-01-01,6389
+out_eia923__boiler_fuel,2008-02-01,6358
+out_eia923__boiler_fuel,2008-03-01,6435
+out_eia923__boiler_fuel,2008-04-01,6443
+out_eia923__boiler_fuel,2008-05-01,6455
+out_eia923__boiler_fuel,2008-06-01,6457
```

Comment:
Monthly partitions will provide a lot of granular information about where changes have happened, but using only unique values of a single column seems like it may not be very adaptable, e.g. in hourly tables, or tables where we need to select on more than one column to get well meaningful chunks, and the number of groups we're tracking in this system will be totally unmanageable by hand -- it'll have to be entirely automatable (with manual review / approval).

If we want to get much more granular data regression / diff testing than what we've been doing with the row counts, then we may want to revisit the more generalized database diff work that @rousik was doing before he vanished, especially now that he's back.